src/tests/test_raft_repl_dev.cpp

```diff
@@ -878,14 +878,6 @@ TEST_F(RaftReplDevTest, BaselineTest) {
     LOGINFO("Homestore replica={} setup completed", g_helper->replica_num());
     g_helper->sync_for_test_start();
 
```

Comment:
We need this as we are not truncating when raft calls compact

Reply:
let`s first make a decision on this topic
https://github.com/eBay/HomeStore/pull/515/files/c3e0ab3f3210c42a63b6b5f6a97386115f1126a9#r1724163097

then we can revisit this. if we need do a real truncate, then this can be removed